<?xml version="1.0" encoding="UTF-8"?><testsuite errors="13" failures="3" name="Set de pasar plata en linea" skipped="0" tests="16" time="5415.47" timestamp="2024-07-19 11:56:21"><testcase name="CP03600M_SYS_Validar proceso por la opción a otro banco en línea a un daviplata"><failure message="No pudo interactuar con el elemento: //XCUIElementTypeStaticText[@name='Por Transfiya'] | //XCUIElementTypeButton[@name='Por Transfiya']" type="java.lang.AssertionError">No pudo interactuar con el elemento: //XCUIElementTypeStaticText[@name='Por Transfiya'] | //XCUIElementTypeButton[@name='Por Transfiya']</failure><system-err>No pudo interactuar con el elemento: //XCUIElementTypeStaticText[@name='Por Transfiya'] | //XCUIElementTypeButton[@name='Por Transfiya']
daviplata.nacional.iOS.utilidades.UtilidadesTCS.clicElement(UtilidadesTCS.java:92)
daviplata.nacional.iOS.steps.PasarPlataSteps.realizarFlujoPasarPlataTransfiya(PasarPlataSteps.java:1617)
daviplata.nacional.iOS.definitions.PasarPlataDefinitions.realizoFlujoPasarPlataTransfiya(PasarPlataDefinitions.java:504)
cucumber.runner.Runner.runPickle(Runner.java:80)
daviplata.nacional.iOS.runners.RunnerPersonalizado.run(RunnerPersonalizado.java:63)
org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)
org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
</system-err></testcase><testcase name="CP03610M_SYS_Validar proceso por la opción a otro banco en línea a un daviplata y realizando aceptacion"><failure message="No se pudo interactuar con el elemento: //XCUIElementTypeButton[@name='Menu de conocimiento']" type="java.lang.AssertionError">No se pudo interactuar con el elemento: //XCUIElementTypeButton[@name='Menu de conocimiento']</failure><system-err>No se pudo interactuar con el elemento: //XCUIElementTypeButton[@name='Menu de conocimiento']
daviplata.nacional.iOS.utilidades.UtilidadesTCS.validateElementVisibility(UtilidadesTCS.java:301)
daviplata.nacional.iOS.steps.LoginSteps.ingresarAApp(LoginSteps.java:129)
daviplata.nacional.iOS.definitions.LoginDefinitions.ingresoUsuarioYContrasena(LoginDefinitions.java:45)
✽.ingreso usuario y contrasena "CC" "10333040" "1234"(src/test/resources/features/PasarPlataEnLinea.feature:28)
</system-err></testcase><testcase name="CP03620M_SYS_Validar proceso por la opción a otro banco en línea a un daviplata con daviplata origen GMF"><error type="java.lang.reflect.InvocationTargetException"/><system-err>null
io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:209)
io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:231)
io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:38)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:84)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:94)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.iniciarAppiumDriver(CustomAppiumDriver.java:40)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.&lt;init&gt;(CustomAppiumDriver.java:18)
daviplata.nacional.iOS.definitions.Hooks.iniciarTests(Hooks.java:41)
cucumber.runner.UnskipableStep.executeStep(UnskipableStep.java:22)
cucumber.runner.Runner.runPickle(Runner.java:80)
daviplata.nacional.iOS.runners.RunnerPersonalizado.run(RunnerPersonalizado.java:63)
org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)
org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
</system-err></testcase><testcase name="CP03650M_SYS_Validar proceso por la opción a otro banco en línea de pedir plata"><error type="java.lang.reflect.InvocationTargetException"/><system-err>null
io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:209)
io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:231)
io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:38)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:84)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:94)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.iniciarAppiumDriver(CustomAppiumDriver.java:40)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.&lt;init&gt;(CustomAppiumDriver.java:18)
daviplata.nacional.iOS.definitions.Hooks.iniciarTests(Hooks.java:41)
cucumber.runner.UnskipableStep.executeStep(UnskipableStep.java:22)
cucumber.runner.Runner.runPickle(Runner.java:80)
daviplata.nacional.iOS.runners.RunnerPersonalizado.run(RunnerPersonalizado.java:63)
org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)
org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
</system-err></testcase><testcase name="CP03660M_SYS_Validar proceso por la opción a otro banco en línea de pedir plata con cliente origen GMF"><error type="java.lang.reflect.InvocationTargetException"/><system-err>null
io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:209)
io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:231)
io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:38)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:84)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:94)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.iniciarAppiumDriver(CustomAppiumDriver.java:40)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.&lt;init&gt;(CustomAppiumDriver.java:18)
daviplata.nacional.iOS.definitions.Hooks.iniciarTests(Hooks.java:41)
cucumber.runner.UnskipableStep.executeStep(UnskipableStep.java:22)
cucumber.runner.Runner.runPickle(Runner.java:80)
daviplata.nacional.iOS.runners.RunnerPersonalizado.run(RunnerPersonalizado.java:63)
org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)
org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
</system-err></testcase><testcase name="CP03670M_SYS_Validar proceso por la opción a otro banco en línea de pedir plata realizando proceso de aceptacion."><failure message="No se pudo interactuar con el elemento: //XCUIElementTypeButton[@name='Menu de conocimiento']" type="java.lang.AssertionError">No se pudo interactuar con el elemento: //XCUIElementTypeButton[@name='Menu de conocimiento']</failure><system-err>No se pudo interactuar con el elemento: //XCUIElementTypeButton[@name='Menu de conocimiento']
daviplata.nacional.iOS.utilidades.UtilidadesTCS.validateElementVisibility(UtilidadesTCS.java:301)
daviplata.nacional.iOS.steps.LoginSteps.ingresarAApp(LoginSteps.java:129)
daviplata.nacional.iOS.definitions.LoginDefinitions.ingresoUsuarioYContrasena(LoginDefinitions.java:45)
✽.ingreso usuario y contrasena "CC" "10333040" "1234"(src/test/resources/features/PasarPlataEnLinea.feature:108)
</system-err></testcase><testcase name="CP03680M_SYS_Validar proceso por la opción a otro banco en línea de pedir plata realizando proceso de rechazo."><error type="java.lang.reflect.InvocationTargetException"/><system-err>null
io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:209)
io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:231)
io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:38)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:84)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:94)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.iniciarAppiumDriver(CustomAppiumDriver.java:40)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.&lt;init&gt;(CustomAppiumDriver.java:18)
daviplata.nacional.iOS.definitions.Hooks.iniciarTests(Hooks.java:41)
cucumber.runner.UnskipableStep.executeStep(UnskipableStep.java:22)
cucumber.runner.Runner.runPickle(Runner.java:80)
daviplata.nacional.iOS.runners.RunnerPersonalizado.run(RunnerPersonalizado.java:63)
org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)
org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
</system-err></testcase><testcase name="CP03690M_SYS_Validar proceso por la opción a otro banco en línea de pedir plata con cliente origen GMF y realizar rechazo"><error type="java.lang.reflect.InvocationTargetException"/><system-err>null
io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:209)
io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:231)
io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:38)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:84)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:94)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.iniciarAppiumDriver(CustomAppiumDriver.java:40)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.&lt;init&gt;(CustomAppiumDriver.java:18)
daviplata.nacional.iOS.definitions.Hooks.iniciarTests(Hooks.java:41)
cucumber.runner.UnskipableStep.executeStep(UnskipableStep.java:22)
cucumber.runner.Runner.runPickle(Runner.java:80)
daviplata.nacional.iOS.runners.RunnerPersonalizado.run(RunnerPersonalizado.java:63)
org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)
org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
</system-err></testcase><testcase name="CP03700M_SYS_Validar proceso donde el destino cuando se le pide dinero, y el origen no tiene suficiente en el disponible."><error type="java.lang.reflect.InvocationTargetException"/><system-err>null
io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:209)
io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:231)
io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:38)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:84)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:94)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.iniciarAppiumDriver(CustomAppiumDriver.java:40)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.&lt;init&gt;(CustomAppiumDriver.java:18)
daviplata.nacional.iOS.definitions.Hooks.iniciarTests(Hooks.java:41)
cucumber.runner.UnskipableStep.executeStep(UnskipableStep.java:22)
cucumber.runner.Runner.runPickle(Runner.java:80)
daviplata.nacional.iOS.runners.RunnerPersonalizado.run(RunnerPersonalizado.java:63)
org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)
org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
</system-err></testcase><testcase name="CP03710M_SYS_Validar que al hacer tap en el icono que está al lado de link tip (A otro banco en línea) el sistema muestra mensaje con una descripción y botón aceptar"><error type="java.lang.reflect.InvocationTargetException"/><system-err>null
io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:209)
io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:231)
io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:38)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:84)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:94)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.iniciarAppiumDriver(CustomAppiumDriver.java:40)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.&lt;init&gt;(CustomAppiumDriver.java:18)
daviplata.nacional.iOS.definitions.Hooks.iniciarTests(Hooks.java:41)
cucumber.runner.UnskipableStep.executeStep(UnskipableStep.java:22)
cucumber.runner.Runner.runPickle(Runner.java:80)
daviplata.nacional.iOS.runners.RunnerPersonalizado.run(RunnerPersonalizado.java:63)
org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)
org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
</system-err></testcase><testcase name="CP03720M_SYS_Validar que el sistema muestra la lista de pagos frecuente en el link A otro banco en línea en Abonos frecuentes"><error type="java.lang.reflect.InvocationTargetException"/><system-err>null
io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:209)
io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:231)
io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:38)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:84)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:94)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.iniciarAppiumDriver(CustomAppiumDriver.java:40)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.&lt;init&gt;(CustomAppiumDriver.java:18)
daviplata.nacional.iOS.definitions.Hooks.iniciarTests(Hooks.java:41)
cucumber.runner.UnskipableStep.executeStep(UnskipableStep.java:22)
cucumber.runner.Runner.runPickle(Runner.java:80)
daviplata.nacional.iOS.runners.RunnerPersonalizado.run(RunnerPersonalizado.java:63)
org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)
org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
</system-err></testcase><testcase name="CP03640M_SYS_Validar proceso por la opción a otro banco en línea a un daviplata y realizando rechazo"><error type="java.lang.reflect.InvocationTargetException"/><system-err>null
io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:209)
io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:231)
io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:38)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:84)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:94)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.iniciarAppiumDriver(CustomAppiumDriver.java:40)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.&lt;init&gt;(CustomAppiumDriver.java:18)
daviplata.nacional.iOS.definitions.Hooks.iniciarTests(Hooks.java:41)
cucumber.runner.UnskipableStep.executeStep(UnskipableStep.java:22)
cucumber.runner.Runner.runPickle(Runner.java:80)
daviplata.nacional.iOS.runners.RunnerPersonalizado.run(RunnerPersonalizado.java:63)
org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)
org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
</system-err></testcase><testcase name="CP037302M_SYS_Validar que al dar tap en el boton (+) Se muestre la pantalla de &quot;que quiere hacer con su plata&quot;"><error type="java.lang.reflect.InvocationTargetException"/><system-err>null
io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:209)
io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:231)
io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:38)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:84)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:94)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.iniciarAppiumDriver(CustomAppiumDriver.java:40)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.&lt;init&gt;(CustomAppiumDriver.java:18)
daviplata.nacional.iOS.definitions.Hooks.iniciarTests(Hooks.java:41)
cucumber.runner.UnskipableStep.executeStep(UnskipableStep.java:22)
cucumber.runner.Runner.runPickle(Runner.java:80)
daviplata.nacional.iOS.runners.RunnerPersonalizado.run(RunnerPersonalizado.java:63)
org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)
org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
</system-err></testcase><testcase name="CP03630M_SYS_Validar proceso por la opción a otro banco en línea a un daviplata y realizando aceptacion con daviplata origen GMF"><error type="java.lang.reflect.InvocationTargetException"/><system-err>null
io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:209)
io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:231)
io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:38)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:84)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:94)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.iniciarAppiumDriver(CustomAppiumDriver.java:40)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.&lt;init&gt;(CustomAppiumDriver.java:18)
daviplata.nacional.iOS.definitions.Hooks.iniciarTests(Hooks.java:41)
cucumber.runner.UnskipableStep.executeStep(UnskipableStep.java:22)
cucumber.runner.Runner.runPickle(Runner.java:80)
daviplata.nacional.iOS.runners.RunnerPersonalizado.run(RunnerPersonalizado.java:63)
org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)
org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
</system-err></testcase><testcase name="CP03730M_SYS_Validar proceso donde el daviplata acepta el pasar plata pero no es abonado ya que tiene el tope de créditos"><error type="java.lang.reflect.InvocationTargetException"/><system-err>null
io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:209)
io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:231)
io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:38)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:84)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:94)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.iniciarAppiumDriver(CustomAppiumDriver.java:40)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.&lt;init&gt;(CustomAppiumDriver.java:18)
daviplata.nacional.iOS.definitions.Hooks.iniciarTests(Hooks.java:41)
cucumber.runner.UnskipableStep.executeStep(UnskipableStep.java:22)
cucumber.runner.Runner.runPickle(Runner.java:80)
daviplata.nacional.iOS.runners.RunnerPersonalizado.run(RunnerPersonalizado.java:63)
org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)
org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
</system-err></testcase><testcase name="CP037301M_SYS_Validar proceso donde el daviplata acepta el pedir plata pero no es abonado ya que tiene el tope de débitos"><error type="java.lang.reflect.InvocationTargetException"/><system-err>null
io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:209)
io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:231)
io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:38)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:84)
io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:94)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.iniciarAppiumDriver(CustomAppiumDriver.java:40)
daviplata.nacional.iOS.utilidades.CustomAppiumDriver.&lt;init&gt;(CustomAppiumDriver.java:18)
daviplata.nacional.iOS.definitions.Hooks.iniciarTests(Hooks.java:41)
cucumber.runner.UnskipableStep.executeStep(UnskipableStep.java:22)
cucumber.runner.Runner.runPickle(Runner.java:80)
daviplata.nacional.iOS.runners.RunnerPersonalizado.run(RunnerPersonalizado.java:63)
org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)
org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
</system-err></testcase></testsuite>